lux/compiler/Compiler.py

```diff
@@ -136,6 +141,35 @@ def expandUnderspecified(ldf, viewCollection):
 						chartTitle = spec.value
 					view.title = f"{spec.attribute} {spec.filterOp} {chartTitle}"
 		return views
+
+	@staticmethod
+	def removeAllInvalid(viewCollection):
+		"""
+		Given an expanded view collection, remove all views that are invalid.
```

Comment:
Add an explanation regarding what types of views are considered invalid and what we are removing here.

Reply:
Added to a new commit.